src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/WebClientConfiguration.kt

```diff
@@ -60,7 +60,6 @@ class WebClientConfiguration(
 
   private fun createAuthorizedWebClient(clientManager: OAuth2AuthorizedClientManager, baseUrl: String): WebClient {
     val oauth2Client = ServletOAuth2AuthorizedClientExchangeFilterFunction(clientManager)
-    oauth2Client.setDefaultClientRegistrationId("interventions-client")
```

Comment:
this is really the crux of this commit

Reply:
_nit_ probably worth extracting `interventions-client` to a constant